testsuite/pytests/test_sp/test_sp_manager.py

```diff
@@ -141,11 +142,10 @@ def test_synapse_creation(self):
                 nest.Simulate(10.0)
                 status = nest.GetStatus(neurons, 'synaptic_elements')
                 for st_neuron in status:
-                    self.assertEqual(10, st_neuron['SE1']['z_connected'])
-                    self.assertEqual(10, st_neuron['SE2']['z_connected'])
+                    assert 10 == st_neuron['SE1']['z_connected']
+                    assert 10 == st_neuron['SE2']['z_connected']
```

Comment:
On second thought, I think the following would be nicer to read:
```suggestion
                    assert st_neuron['SE1']['z_connected'] == 10
                    assert st_neuron['SE2']['z_connected'] == 10
```